### DISCUSSION TITLE
Support library panels in v7

### DISCUSSION THREAD

**Discussion by wenqi73** [grafana/grafana#60503] [Dashboards, Panels & Visualizations]
Hi guys, I see that this is already supported in 8.5(https://grafana.com/docs/grafana/v8.5/panels/library-panels/), is there a plan to support it also on v7(the Apache-2.0 license)? Or accept a PR from the community?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
